client/src/pages/Profile/Profile.css

```diff
@@ -0,0 +1,59 @@
+/* .FeatureContainer_image.User {
+  background-image: url('../../assets/images/hairpink.jpg');
+}
+.FeatureContainer_image.Owner {
+  background-image: url('../../assets/images/hairgreysm.jpg');
+} */
+
+.FeatureContainer_image {
+  background-image: url('../../assets/images/hairgreysm.jpg');
+}
+
+.Profile_user {
+  text-align: center;
+  grid-column: 5/11;
+  margin-top: 125px;
+}
+
+.Profile_name h1 {
+  text-align: center;
+}
+.user-pix-placeholder {
+  width: 180px;
+  height: 180px;
+  border-radius: 50%;
+  background: rgb(216, 212, 212);
+  padding: 70px;
+  margin: auto;
+  margin-top: -20px;
+}
+.user-reviews-placeholder {
+  margin-top: 100px;
+  font-size: 1.6rem;
+}
```

Comment:
Hi @cynthiaeddy!  You may have missed converting these classes to the new format, BEM - React Style,  agreed upon by the team.  I attached a picture of the syntax for convenience.  See below. 

![image](https://user-images.githubusercontent.com/32943879/162635224-a93c6dde-44eb-42e0-a01d-d407ca52c1b5.png)

Reply:
yeah i know the above isn't BEM - React style - it's only temporary css until it's replaced with the actual components, images - that's why i didn't use BEM.